src/myna/core/workflow/config.py

```diff
@@ -294,6 +319,34 @@ def config(input_file, output_file=None, show_avail=False, overwrite=False):
             build_struct = (
                 os.path.abspath(case_dir).replace(base_path, "").split(os.sep)
             )
+            if "build_region" in step_obj.types:
+                build_region = build_struct[2]
+                build_region_parts = data_dict_case["build"]["build_regions"][
+                    build_region
+                ]["parts"]
+                keys = list(data_dict_case["build"]["build_regions"].keys())
+                for key in keys:
+                    if key != build_region:
+                        data_dict_case["build"]["build_regions"].pop(key, None)
+                    else:
+                        for part in data_dict_case["build"]["parts"]:
+                            if part not in build_region_parts:
+                                data_dict_case["build"]["parts"].pop(part, None)
+                            elif "layer" in step_obj.types:
+                                layer = build_struct[3]
+                                keys = list(
+                                    data_dict_case["build"]["parts"][part]
+                                    .get("layer_data", {})
+                                    .keys()
+                                )
+                                for key in keys:
+                                    if int(key) != int(layer):
+                                        data_dict_case["build"]["parts"][part][
+                                            "layer_data"
+                                        ].pop(key, None)
+                                data_dict_case["build"]["parts"][part]["layers"] = [
+                                    int(layer)
+                                ]
             if "part" in step_obj.types:
```

Comment:
You should be able to. I should test that 🤔

Reply:
Turns out that the way things were set up, some weird behavior happens if you try that. I don't like some of the things that happen with that, so I'm reworking `config.py` a bit to fix. Changes to come.

Reply:
Changes implemented. I had initially tried to work around to avoid adding `build_region` as another top-level hierarchy, but that didn't work.

"Propertly" implementing `build_region` as the top-level category creates another branch for most of the conditionals in `config.py`, which is getting convoluted...but using some of the dictionary utilities **somewhat** helps condense the code.